index.html

```diff
@@ -232,7 +232,7 @@ <h2>Introduction</h2>
 
       <p>
 This specification provides a standard way to express <a>credentials</a> on the
-Web in a way that is cryptographically secure, privacy respecting, and
+Web, a way that is cryptographically secure, privacy respecting, and
```

Comment:
Hmm, I feel like the meaning of the sentence changed w/ this modification. The specification isn't providing two things. It's providing one thing. I can kinda sorta see the old meaning in the new text, but I feel like it could be misinterpreted by someone not carefully reading the sentence. 

Suggest to revert to the old language.

Reply:
Done.